src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -200,7 +200,7 @@ public function testValidateCustomErrorPath()
         $this->buildViolation('myMessage')
             ->atPath('property.path.bar')
             ->setParameter('{{ value }}', '"Foo"')
-            ->setInvalidValue('Foo')
+            ->setInvalidValue($entity2)
```

Comment:
could this be a BC break? To not have the field value as the invalid value anymore but the entity itself?